source/extensions/omni.isaac.lab/omni/isaac/lab/envs/manager_based_env.py

```diff
@@ -87,11 +87,22 @@ def __init__(self, cfg: ManagerBasedEnvCfg):
         print("[INFO]: Base environment:")
         print(f"\tEnvironment device    : {self.device}")
         print(f"\tPhysics step-size     : {self.physics_dt}")
-        print(f"\tRendering step-size   : {self.physics_dt * self.cfg.sim.substeps}")
+        print(f"\tRendering step-size   : {self.physics_dt * self.cfg.sim.render_interval}")
         print(f"\tEnvironment step-size : {self.step_dt}")
         print(f"\tPhysics GPU pipeline  : {self.cfg.sim.use_gpu_pipeline}")
         print(f"\tPhysics GPU simulation: {self.cfg.sim.physx.use_gpu}")
 
+        if self.cfg.sim.render_interval < self.cfg.decimation:
+            msg = (
+                f"The render interval ({self.cfg.sim.render_interval}) is smaller than the decimation "
+                f"({self.cfg.decimation}). Multiple multiple render calls will happen for each environment step."
+                "If this is not intended, set the render interval to be equal to the decimation."
+            )
+            carb.log_warn(msg)
+
+        # counter for simulation steps
+        self._step_counter = 0
```

Comment:
Can we rename to be clearer?

```suggestion
        self._sim_step_counter = 0
```

Also I noticed the simulation context has a variable already for this:
`self.sim.current_time_step_index`.